google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -35,11 +35,13 @@ StatusOr<rest_internal::RestRequest> BuildRestRequest(GetJobRequest const& r,
     return internal::InvalidArgumentError(
         "Invalid GetJobRequest: Job Id is empty", GCP_ERROR_INFO());
   }
-  // Fix the endpoints prefix and suffixes.
-  std::string endpoint = "https://bigquery.googleapis.com/";
-  if (opts.has<EndpointOption>()) {
-    endpoint = opts.get<EndpointOption>();
+  if (!opts.has<EndpointOption>()) {
```

Comment:
nit: I am not convinced this is necessary, but cannot say that it hurts anything.